composer.json

```diff
@@ -8,14 +8,37 @@
     "typo3/cms-adminpanel": "9.5.*@dev"
   },
   "require-dev": {
-    "roave/security-advisories": "dev-master"
+    "roave/security-advisories": "dev-master",
+    "typo3/cms-install": "9.5.*@dev",
+    "typo3/cms-extensionmanager": "9.5.*@dev",
+    "typo3/cms-recordlist": "9.5.*@dev"
   },
   "replace": {
-    "introduction": "*"
+    "adminpanel_extended": "self.version"
   },
   "autoload": {
     "psr-4": {
       "Psychomieze\\AdminpanelExtended\\": "Classes/"
     }
+  },
+  "config": {
+    "vendor-dir": ".Build/vendor",
+    "bin-dir": ".Build/bin"
+  },
+  "extra": {
+    "typo3/cms": {
+      "extension-key": "adminpanel_extended",
+      "cms-package-dir": "{$vendor-dir}/typo3/cms",
+      "app-dir": ".Build",
+      "web-dir": ".Build/web"
+    }
+  },
+  "scripts": {
+    "create-extension-directory": "mkdir -p .Build/web/typo3conf/ext",
+    "link-package": "ln -snvf ../../../../. .Build/web/typo3conf/ext/adminpanel_extended",
```

Comment:
nope, please don't use platform specific commands.

Reply:
Fair enough, but how would one then accomplish these things? Using a composer script class that is listening on events seems to be overkill.

Reply:
After thinking about it, a composer-script-event-class would be the solution. I'll introduce one later.